src/hapi/pipelines/configs/core.yaml

```diff
@@ -38,3 +38,111 @@ org_type:
   resource: "Organization Types (Beta) - CSV"
   format: "csv"
   headers: 2
+
+sector_map:
+  abna: "SHL"
```

Comment:
I worry that it will be tricky to manage such a long growing list. I guess the important thing is to make sure we keep it alphabetically sorted (there are tools out there that can help with this, although none that I could find for pre-commit). 

I was thinking about other ways it could be broken down or organized. They may be terrible suggestions so let me know what you think:
 
1. Make a separate sector_map for each country. The drawback is that there may be duplicates that would lead to repetition (and possible inconsistencies / errors) in the config files. 
2. Flip the key-values in the config file - i.e. for each sector type, list all the possible variations. Then in the code we'd have to invert it, while watching out for duplicates, which would be the big issue.